adafruit_ble_radio.py

```diff
@@ -167,11 +167,14 @@ def receive_full(self):
         * a microsecond timestamp: the value returned by time.monotonic() when
           the message was received.
 
+        :param float wait: The length of time the radio listens for a
```

Comment:
I think it would be good to specify the units of time in this docstring so mention that it's measured in seconds.

Additionally, the sphinx docs did not build successfully. It looks like possibly there is some issue with the way this docstring is formatted.

It can be helpful to run the docs build locally when attempting to troubleshoot the specific issue. This learn guide section covers how to install sphinx and use it to build the docs: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/sharing-our-docs-on-readthedocs#sphinx-2615853-1

Reply:
This is a bit trickier for me to resolve, because my local build of the docs fails with the error:
`autodoc: failed to import module 'adafruit_ble_radio'; the following exception was raised:
No module named 'micropython'`

I was able to setup the webhook to readthedocs (for the most part, I see that the actions are synchronized to the Adafruit branch, so I'm sorry for the extra noise), and it seems like all is well when I preview the docs: [https://mlupo-adafruit-circuitpython-ble-radio.readthedocs.io/en/latest/api.html#adafruit_ble_radio.Radio.receive_full](https://mlupo-adafruit-circuitpython-ble-radio.readthedocs.io/en/latest/api.html#adafruit_ble_radio.Radio.receive_full). But, I'm not sure if it will pass all checks.

Reply:
I noticed as well that the error for sphinx locally was different from the actions. I'm not entirely sure what is leading to that difference. Looks like you got it straightened out anyhow though.